src/foolscap/test/test_schema.py

```diff
@@ -53,43 +55,43 @@ def testLargeInteger(self):
 
     def testByteString(self):
         c = schema.ByteStringConstraint(10)
-        self.conforms(c, "I'm short")
-        self.violates(c, "I am too long")
-        self.conforms(c, "a" * 10)
-        self.violates(c, "a" * 11)
+        self.conforms(c, six.b("I'm short"))
+        self.violates(c, six.b("I am too long"))
```

Comment:
can't patterns like this all be like `self.conforms(c, b"I'm short")` instead ..?

Reply:
six.b(...) makes it explicit that this is a bytes (3)/ str(2) type.

Reply:
Doesn't `b"foo"` make it explicit also?

Reply:
Yes it does, Its a question of style. I often find searching for six.b('...") convenient when editing, than b'...' in my editor (Emacs).